src/color-scale/color-scale.js

```diff
@@ -153,6 +166,9 @@ const Self = class ColorScale extends NudeElement {
 			additionalDependencies: ["info"],
 		},
 		info: {},
+		vs: {
+			type: Color,
```

Comment:
Ah I see why you're doing the `#hasVs` thing. That’s not the right way; it means that if someone sets it via the property you get a different behavior. You want two properties: one for the raw value, and one for the parsed value, which is either `true` (previous color), `false` (no vs), or a `Color` instance.

And yes, this is another case of `rawProp`.

Reply:
Yes, I'm doing this precisely because I can't see another way to distinguish different cases: color, no color, no vs.

Reply:
Another option would be to use a custom parse function. Up to you.

Reply:
I wonder if it may be useful to provide a way to do vs with the previous or next color. In that case we could have `vs="previous"` and `vs="next"` which may be more explanatory than a bare `vs` with no value.

Reply:
> Another option would be to use a custom parse function.

Yeah, I was going to try this later. Glad it makes sense. :)

Reply:
> I wonder if it may be useful to provide a way to do vs with the previous or next color. In that case we could have `vs="previous"` and `vs="next"` which may be more explanatory than a bare `vs` with no value.

I like it